models/models-games.js

```diff
@@ -10,4 +11,14 @@ exports.selectReviewByID = (review_id) => {
     return db.query(`SELECT * FROM reviews WHERE review_id = $1`, [review_id]).then((review) => {
         return review.rows[0]
     })
-};
\ No newline at end of file
+};
+
+exports.updateReviewByID = (review_id, votes) => {
+    return db.query('UPDATE reviews SET votes = votes + $1 WHERE review_id = $2', [votes, review_id])
+        .then(() => {
+            return db.query('SELECT * FROM reviews WHERE review_id = $1', [review_id])
+        }).then((review) => {
+            return review.rows[0]
+
+        })
```

Comment:
Omg, its so funny you say that because I am pretty sure that I complained about needing to do another .then someone explained to me last time we had to do a Patch that I can just instantly return it in the query before........ and SOMEHOW I went ahead and instantly forgot about it again? 🤦 
Thanks so much, eventually I'll remember hahaha